### PR TITLE
msi_z690/faq.md: add section with description of problems with gpu

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemyslaw Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/variants/msi_z690/faq.md [hunk 1]
```diff
@@ -89,3 +89,29 @@ ones in Dasharo firmware (provied by the
...
+`Legacy Option ROMs` are only supported on legacy BIOS, such as SeaBIOS.
+Legacy BIOS checks the availability of `Option ROM` and if its signature
+matches, it executes its entry point. This option ROM initializes the
+graphics.The only way to support `Option ROM`  in UEFI is through CSM,
+which we do not have implemented.
+
+`EFI Option ROMs` are nothing more than EFI drivers which have the same form
+as the UEFI files (PE format). UEFI firmware scans the `Option ROM` space of
+the graphics card and if it finds a potential `EFI Option ROM` with PE
+signature, it executes the option. This option ROM initializes the
```
Comment (miczyg1) [inline docs/variants/msi_z690/faq.md:110]:
```suggestion
signature, it executes the file. This option ROM initializes the
```

File: docs/variants/msi_z690/faq.md [hunk 2]
```diff
@@ -89,3 +89,29 @@ ones in Dasharo firmware (provied by the
...
+Due to the fact, that there's no possibility to insert all available GPU
+drivers into the firmware, the solution in the form of the `Option ROM` is in
+use. `Option ROMs` are the drivers flashed in the GPUs non-volatile memory.
+These types of drivers can be divided into `Legacy Option ROMs` and
+`EFI Option ROMs`.
+
+`Legacy Option ROMs` are only supported on legacy BIOS, such as SeaBIOS.
+Legacy BIOS checks the availability of `Option ROM` and if its signature
+matches, it executes its entry point. This option ROM initializes the
+graphics.The only way to support `Option ROM`  in UEFI is through CSM,
```
Comment (miczyg1) [inline docs/variants/msi_z690/faq.md:104]:
```suggestion
graphics. The only way to support `Option ROM`  in UEFI is through CSM,
```

File: docs/variants/msi_z690/faq.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/msi_z690/faq.md]: improved

File: docs/variants/msi_z690/faq.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/msi_z690/faq.md]: improved

**Pull request merged**